roles/config-rh-sso/templates/id-providers.json.j2

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We should look to provide the appropriate `mandatory` and `default` filters to this